chainstate/src/detail/transaction_verifier/tx_index_cache.rs

```diff
@@ -0,0 +1,187 @@
+use std::collections::{btree_map::Entry, BTreeMap};
```

Comment:
copyright

Reply:
Oh, crap...

Reply:
I wonder why the python script didn't figure it out